user_tools/src/spark_rapids_pytools/resources/dataproc-configs.json

```diff
@@ -210,6 +210,28 @@
               "    --worker-accelerator type=nvidia-tesla-{},count={}"
             ]
           }
+        },
+        {
+          "sectionID": "gpuClusterCreationScript",
+          "content": {
+            "header": [
+              "",
+              "To create a GPU cluster, run the following script:",
+              ""
+            ]
+          }
+        },
+        {
+          "sectionID": "runUserToolsBootstrap",
+          "content": {
+            "header": [
+              "",
+              "Once the cluster is created, run the Bootstrap tool to provide optimized",
+              "RAPIDS Accelerator for Apache Spark configs based on GPU cluster shape.",
+              "Note that the requires SSH access in order to override the Apache Spark default configurations.",
```

Comment:
typo: `Note that the ? requires`

also does the new user tools require SSH or should we let them know about bootstrap local mode if they don't have SSH access?

Reply:
I thought about that as well. My intuition was that the feature is to show them how to bootstrap which means that they have to SSH.
I can append a note reminding the user that he can dump the configurations without applying the changes to the cluster.